src/asInput/asInput.scss

```diff
@@ -3,8 +3,8 @@
 @import "~bootstrap/scss/utilities/_screenreaders.scss";
 @import '~bootstrap/scss/input-group';
 
-.fa-icon-spacing {
-    padding: 0px 5px 0px 0px;
```

Comment:
Curious about this CSS change as well, as it doesn't look to me to be a simple swap.

Reply:
Nevermind, I started from the bottom up and made it to the new scss file that this was moved to :)